scanner/keppel/scanner/scanner.go

```diff
@@ -156,6 +156,28 @@ func (s *Scanner) ListManifests(account string, repository string) ([]models.Man
 	return manifestResponse.Manifests, nil
 }
 
+func (s *Scanner) ListManifestsOfManifest(account string, repository string, manifest string) ([]models.Manifest, error) {
```

Comment:
As pointed out a while ago, we need more comments. Without having the full context why this PR was created at all, I might need additional information (in the code) which explains why this method (in this case ListManifestsOfManifest) is needed at all.

Reply:
done